django_mongodb/compiler.py

```diff
@@ -136,7 +136,12 @@ def build_query(self, columns=None):
         self.check_query()
         self.setup_query()
         query = self.query_class(self, columns)
-        query.add_filters(self.query.where)
+        try:
+            query.mongo_query = self.query.where.as_mql(self, self.connection)
+        except FullResultSet:
+            query.mongo_query = {}
+
+        # query.add_filters(self.query.where)
```

Comment:
You can remove it rather than comment it out. You can similarly remove `add_filters()` and all the code associated with it.

Reply:
Done